scenes/menus/title/main_menu/main_menu.gd

```diff
@@ -82,14 +83,14 @@ func _process(delta):
 			center_pos_idx = 2
 			
 			var items = [[story, selector_story], [settings, selector_settings], [extra, selector_extra], [ld, selector_ld]]
-			# Be sure to initialize num_items before step() -> _clamp_cycle_step() reads it.
+			# Be sure to initialize num_items before _cycle_through() -> _clamp_cycle_step() reads it.
```

Comment:
Change comment to say `_cycle_through() -> _cycle_increment()`.

Reply:
Whoops, didn't see that, I'll fix that on the master branch.

Reply:
Nevermind, it's fine. I disliked the name `_cycle_increment()` so I changed it. Apologies, I misread your note.